### PR TITLE
Fix: fix broken links by replacing with updated links in actionRespon…

### PR DESCRIPTION
**Pull request by adeyinkaoresanya**
…se.js   Signed-off-by: Adeyinka Oresanya adeyinkaoresanya@gmail.com

This PR...

## Changes

-

## Screenshots

(prefer animated gif)

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

Fixes #

I replaced broken links [https://chaoss.community/participate, https://chaoss.community/kbtopic/working-groups/] with the appropriate updated links

### CODE REVIEW

File: components/actions/actionResponses.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ async function contribute(say) {
 
 async function helpWithWebsite(say) {
   return await say(
-    `You clicked *Help with the Website*\n first step is getting to know our community! You can connect with us in any of the ways described in our Participate page here: https://chaoss.community/participate.
+    `You clicked *Help with the Website*\n first step is getting to know our community! You can connect with us in any of the ways described in our Participate page here: https://chaoss.community/kb-getting-started/.
```
Comment (misspee007) [inline components/actions/actionResponses.js:27]: Let's replace "Participate page" with "Getting Started page".

**Review by misspee007 — CHANGES_REQUESTED** [review]
This looks great, just one minor change and we can merge. I've left a comment on the line concerned. Thanks for contributing!

File: components/actions/actionResponses.js (hunk 1 above)
Comment (adeyinkaoresanya) [reply components/actions/actionResponses.js]: Done!

Comment (misspee007) [reply components/actions/actionResponses.js]: Awesome. Thanks for contributing!

**Pull request merged**